src/jtag/drivers/ftdi.c

```diff
@@ -95,6 +108,29 @@ static uint8_t ftdi_jtag_mode = JTAG_MODE;
 
 static bool swd_mode;
 
+#if BUILD_FTDI_OSCAN1 == 1
+/*
+  cJTAG OSCAN1, in lieu of 4-wire JTAG, is starting to be a configuration option
+  for some SiFive hardware. An FTDI-based adapter that can be configured to drive
```

Comment:
If possible, can you emphasize that this is not some SiFive-specific invention, but there may be other hardware out there that this code is useful for?